### PR TITLE
refactor(dht): eslint dht import

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Enables all eslint import rules

### Issues

- based on https://github.com/gradido/gradido/pull/3042

### Todo

- [X] None

### CODE REVIEW

File: .github/workflows/test_dht_node.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         uses: actions/checkout@v3
       
       - name: Lint
-        run: cd dht-node && yarn && yarn run lint
+        run: cd database && yarn && cd ../backend && yarn && yarn run lint
```
Comment (Elweyn) [inline .github/workflows/test_dht_node.yml:54]:
```suggestion
        run: cd dht-node && yarn && yarn run lint && cd ../database && yarn && yarn run lint && cd ../backend && yarn && yarn run lint
```

**Review by Elweyn — CHANGES_REQUESTED** [review]
Not totally sure if the suggestion is appropriate but I think that if you want to lint you lint everything: dht, backend and database

File: .github/workflows/test_dht_node.yml (hunk 1 above)
Comment (ulfgebhardt) [reply .github/workflows/test_dht_node.yml:54]: This line makes sure the database has all required `node_modules` and built files so testing/linting works properly.

Only the current project (e.g. `dht_node`) is linted.

Comment (ulfgebhardt) [inline .github/workflows/test_dht_node.yml:54]:
```suggestion
        run: cd database && yarn && cd ../dht-node && yarn && yarn run lint
```

Comment (ulfgebhardt) [reply .github/workflows/test_dht_node.yml]: To you are right - there was a mistake in this like - it should be `cd ../dht-node`

**Review by Elweyn — APPROVED** [review]

**Pull request merged**